raiden/network/transport/matrix/utils.py

```diff
@@ -122,9 +122,11 @@ def stop(self) -> None:
         self._reset_state()
 
     @property
-    def known_addresses(self) -> KeysView[Address]:
+    def known_addresses(self) -> List[Address]:
```

Comment:
`KeysView` is a set-like structure. Therefore returning a set would be better (also the result of this call is often used in `in` tests).

Reply:
This is actually only used for iteration, but I will change the type to set.